main.js

```diff
@@ -0,0 +1,90 @@
+
+var News = document.getElementById('News');   
```

Comment:
**News** should be lowercase.  In JavaScript, you use camelCase normally. If you use Capital, this implies it is a constructor function that we will call with `var something = new News()` for example.